.github/workflows/pr.yaml

```diff
@@ -14,4 +14,13 @@ jobs:
       - name: Helm Lint
         working-directory: ${{ github.workspace }}/src/github.com/${{ github.repository }}
         run: |
-          make helm-lint HELM="docker run --rm -i -v $PWD:/apps infoblox/helm:3.2.4-5b243a2"
+          mkdir -p ~/.kube
+          make helm-lint
+      - uses: helm/kind-action@v1.0.0
+      - name: Test
+        working-directory: ${{ github.workspace }}/src/github.com/${{ github.repository }}
+        run: |
+          kubectl cluster-info
+          make deploy-konk
```

Comment:
Why isn't this using --wait?

Reply:
The static init pod never becomes ready, so `--wait` blocks forever. We can add `--wait` once this issue is resolved.